CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
+
+## [8.0.0] - 2022-10-18
```

Comment:
It's unclear here why we are cutting an 8.0 - wondering if we should add some **BREAKING** lines about the messaging stuff?  I think technically we are making breaking changes by enforcing messaging.